sanitizers/src/main/java/com/code_intelligence/jazzer/sanitizers/RegexInjection.kt

```diff
@@ -49,7 +49,7 @@ object RegexInjection {
     fun compileWithFlagsHook(method: MethodHandle, alwaysNull: Any?, args: Array<Any?>, hookId: Int): Any? {
         val pattern = args[0] as String?
         val hasCanonEqFlag = ((args[1] as Int) and Pattern.CANON_EQ) != 0
-        return hookInternal(method, pattern, hasCanonEqFlag, args, hookId)
+        return hookInternal(method, pattern, hasCanonEqFlag, hookId, args)
```

Comment:
Spread `args`?

Reply:
Fixed, thanks. I think it's not strictly necessary, but definitely more readable that way.